draft-irtf-cfrg-vdaf.md

```diff
@@ -781,44 +785,43 @@ a particular encoding of that quantity as a byte string.
 
 Some common functionalities:
 
-* `zeros(len: int) -> bytes` returns an array of zero bytes. The length of
-  `output` MUST be `len`.
+* `zeros(len: int) -> bytes` returns an array of zero bytes of the requested
+  length (`len`).
 
-* `gen_rand(len: int) -> bytes` returns an array of random bytes generated by a
-  cryptographically secure pseudorandom number generator (CSPRNG). The length
-  of `output` MUST be `len`.
+* `gen_rand(len: int) -> bytes` returns a byte array of the requested length
+  (`len`) generated by a cryptographically secure pseudorandom number generator
+  (CSPRNG).
 
-* `byte(int: int) -> bytes` returns the representation of `int` as a byte
-  string. The value of `int` MUST be in `[0,256)`.
+* `byte(x: int) -> bytes` returns the representation of `x` in range
+  `[0, 256)`.
```

Comment:
I think we should still mention that this is mapping to single-byte byte strings.
```suggestion
* `byte(x: int) -> bytes` returns the representation of the integer `x` in range
  `[0, 256)` as a single-byte byte string.
```